### PR TITLE
L104: Ban GRPC_OP_SEND_STATUS_FROM_SERVER in combination with recv ops

### PR DESCRIPTION
**Pull request by ctiller**

### CODE REVIEW

File: L104-core-ban-recv-with-send-status.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Title
```
Comment (markdroth) [inline L104-core-ban-recv-with-send-status.md:1]: Please change to "L104: C-core: Ban GRPC_OP_SEND_STATUS_FROM_SERVER in combination with recv ops".

File: L104-core-ban-recv-with-send-status.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+----
+* Author(s): ctiller
+* Approver: markdroth
+* Status: In Review
+* Implemented in: C Core
+* Last updated: 2022/11/03
+* Discussion at: <google group thread> (filled after thread exists)
```
Comment (markdroth) [inline L104-core-ban-recv-with-send-status.md:8]: Please create the thread and add a link here.

**Review by markdroth — COMMENTED** [review]
I like this change -- it will get us closer to eventually having a much more idiomatic C-core API.

File: L104-core-ban-recv-with-send-status.md (hunk 1 above)
Comment (ctiller) [reply L104-core-ban-recv-with-send-status.md]: done

File: L104-core-ban-recv-with-send-status.md (hunk 2 above)
Comment (ctiller) [reply L104-core-ban-recv-with-send-status.md]: done

**Review by markdroth — APPROVED** [review]

**Pull request merged**